src/app/components/licenses-modal/licenses-modal.component.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+import { HttpClient } from '@angular/common/http';
+import { isDevMode } from '@angular/core';
+
+@Component({
+  selector: 'app-licenses-modal',
+  templateUrl: './licenses-modal.component.html',
+  styleUrls: ['./licenses-modal.component.scss'],
+})
+export class LicensesModalComponent {
+  licenses = '';
+
+  constructor(
+    private dialogRef: MatDialogRef<LicensesModalComponent>,
+    private httpClient: HttpClient
+  ) {
+    // Licenses file is generated during build time and is available at /3rdpartylicenses.txt
+    // Dev mode uses the file from assets folder assets/example-licenses.txt
+    const licensePath = isDevMode() ? 'assets/example-licenses.txt' : '3rdpartylicenses.txt';
```

Comment:
Could we use `LICENSE` file to replace `assets/example-licenses.txt`?

Reply:
That is a better solution. I have updated the PR with this change.
Thanks!